addons/payment_stripe/controllers/main.py

```diff
@@ -19,9 +18,45 @@
 
 class StripeController(http.Controller):
     _checkout_return_url = '/payment/stripe/checkout_return'
+    _onboarding_return_url = '/payment/stripe/onboarding/return/'
+    _onboarding_refresh_url = '/payment/stripe/onboarding/refresh/'
     _validation_return_url = '/payment/stripe/validation_return'
     WEBHOOK_AGE_TOLERANCE = 10*60  # seconds
 
+    @http.route(
+        _onboarding_return_url + '<int:acquirer_id>/<int:menu_id>', type='http', auth='public'
+    )
+    def stripe_onboarding_return(self, acquirer_id, menu_id):
```

Comment:
```suggestion
    def stripe_return_from_onboarding(self, acquirer_id, menu_id):
```
To be consistent with the other return method.